src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/GlobalSuppressions.cs

```diff
@@ -9,7 +9,3 @@
                 "Reliability",
                 "RS0006:Do not mix attributes from different versions of MEF",
                 Justification = "<Pending>")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("Style", "IDE0030:Use null propagation", Justification = "https://github.com/dotnet/roslyn/issues/161", Scope = "member", Target = "~M:Microsoft.VisualStudio.ProjectSystem.VS.Editor.TempFileTextBufferManager.#ctor(Microsoft.VisualStudio.ProjectSystem.UnconfiguredProject,Microsoft.VisualStudio.ProjectSystem.VS.Editor.IProjectXmlAccessor,Microsoft.VisualStudio.Editor.IVsEditorAdaptersFactoryService,Microsoft.VisualStudio.Text.ITextDocumentFactoryService,Microsoft.VisualStudio.ProjectSystem.VS.UI.IVsShellUtilitiesHelper,Microsoft.VisualStudio.IO.IFileSystem,Microsoft.VisualStudio.ProjectSystem.IProjectThreadingService,System.IServiceProvider)")]
```

Comment:
Funny to see we have 3 different urls in subsequent justifications. https://github.com/dotnet/roslyn/issues/16114 is the only coorect url though.